crypto-analyzer/src/application.py

```diff
@@ -0,0 +1,8 @@
+import plots
+import cryptobot_api
+
+
+bot = cryptobot_api.initialize_cryptonator()
```

Comment:
Den tha evgaze perissoterh logikh an o `cryptonator` xrhsimopoiouse to `cryptobot_api`? Se auti th grammh fainetai to antitheto. Pos xerei o `cryptobot_api` na xekinaei ton `cryptonator`?

Reply:
Mporeis na dwseis ena paradhgma gia to ti 8a mporouse na symperilambanei to `cryptobot_api` giati arxizw na pisteuw oti mallon den to xreiazomai.